clients/privacy-center/components/consent/NoticeDrivenConsent.tsx

```diff
@@ -170,26 +170,21 @@ const NoticeDrivenConsent = () => {
         const notice = notices.find(
           (n) => n.privacy_notice_history_id === historyKey
         );
-        const servedNotice = servedNotices?.find(
-          (sn) => sn.privacy_notice_history?.id === historyKey
-        );
         return { historyKey, preference, notice, servedNotice };
```

Comment:
I think we don't need to pass `servedNotice` here anymore since we removed the assignment and also the `map` right below isn't using it anymore either?

Reply:
yup you're right!